BLUESPAWN-client/src/user/BLUESPAWN.cpp

```diff
@@ -6,8 +6,105 @@
 #include "common/StringUtils.h"
 #include "util/eventlogs/EventLogs.h"
 
+#include "hunt/hunts/HuntT1004.h"
+#include "hunt/hunts/HuntT1037.h"
+#include "hunt/hunts/HuntT1050.h"
+#include "hunt/hunts/HuntT1055.h"
+#include "hunt/hunts/HuntT1060.h"
+#include "hunt/hunts/HuntT1100.h"
+#include "hunt/hunts/HuntT1101.h"
+#include "hunt/hunts/HuntT1103.h"
+#include "hunt/hunts/HuntT1131.h"
+#include "hunt/hunts/HuntT1138.h"
+#include "hunt/hunts/HuntT1182.h"
+#include "hunt/hunts/HuntT1183.h"
+
+#include "monitor/ETW_Wrapper.h"
+
+#include "mitigation/mitigations/MitigateM1042-LLMNR.h"
+#include "mitigation/mitigations/MitigateM1042-WSH.h"
+#include "mitigation/mitigations/MitigateV1093.h"
+#include "mitigation/mitigations/MitigateV1153.h"
+#include "mitigation/mitigations/MitigateV3338.h"
+#include "mitigation/mitigations/MitigateV63597.h"
+#include "mitigation/mitigations/MitigateV63817.h"
+#include "mitigation/mitigations/MitigateV63825.h"
+#include "mitigation/mitigations/MitigateV63829.h"
+#include "mitigation/mitigations/MitigateV72753.h"
+#include "mitigation/mitigations/MitigateV73519.h"
+
 #include <iostream>
 
+IOBase& Bluespawn::io = CLI();
+HuntRegister Bluespawn::huntRecord{ io };
+MitigationRegister Bluespawn::mitigationRecord{ io };
+
+Bluespawn::Bluespawn() {
+	using namespace Hunts;
+
+	HuntT1004* t1004 = new HuntT1004(huntRecord);
+	HuntT1037* t1037 = new HuntT1037(huntRecord);
+	HuntT1050* t1050 = new HuntT1050(huntRecord);
+  HuntT1055* t1055 = new HuntT1055(huntRecord);
+	HuntT1060* t1060 = new HuntT1060(huntRecord);
+	HuntT1100* t1100 = new HuntT1100(huntRecord);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Variable 't1100' is assigned a value that is never used.](https://app.codacy.com/manual/ION28/BLUESPAWN/pullRequest?prid=4960363)